src/components/editMeasure/details/cmsIdentifier/CmsIdentifier.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import {
+  InputLabel,
+  MadieTooltip,
+  Button,
+  ReadOnlyTextField,
+} from "@madie/madie-design-system/dist/react";
+
+export default function CmsIdentifier({ label, cmsId, model, onClick }) {
+  return (
+    <div
+      style={{
+        width: 1,
+        display: "flex",
+        flexDirection: "column",
+        flexGrow: 1,
+        fontFamily: "Rubik",
+        lineHeight: "19px",
+        fontSize: "16px",
+      }}
+    >
+      <div style={{ display: "flex", flexDirection: "row" }}>
+        <InputLabel for="cmsId">{label}</InputLabel>
+
+        <div>
+          <MadieTooltip
+            tooltipText="Once a CMS Identifier has been generated it may not be modified or removed for any draft or version of a measure."
+            id={`cms-id-tooltip`}
+          />
+        </div>
+      </div>
```

Comment:
I think you don't want to display the label with a tooltip. I would suggest moving this to else block on line 45